package br.com.alura.bytebank.exception

class FalhaAutenticacaoException(mensagem: String = "Falha durante a autenticacao!"): Exception(mensagem)